projects/systelab-components/src/lib/contextmenu/context-menu-submenu-item.component.ts

```diff
@@ -0,0 +1,52 @@
+import { Component, ElementRef, Input, QueryList, ViewChildren, } from '@angular/core';
+import { ContextMenuOption } from './context-menu-option';
+import { ContextMenuComponent } from './context-menu.component';
+
+@Component({
+	selector:    'systelab-context-menu-submenu-item',
+	templateUrl: 'context-menu-submenu-item.component.html'
+})
+export class ContextMenuSubmenuItemComponent {
+
+	@Input() public myAction: ContextMenuOption;
```

Comment:
my?

Is the code convention to use my for class properties?

Reply:
It was to identify the action of the submenu respect to the action of the parent, but I will remove the my